client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -10,11 +11,16 @@ const DateControlPickerDate = ( {
 	onApply,
 	onCancel,
 }: DateControlPickerDateProps ) => {
+	const translate = useTranslate();
 	// TODO: Rename component?
 	// Feels a bit confusing now. Should have a better idea
 	// of appropriate names once hierarchy is finalized.
 	return (
 		<div className="date-control-picker-date">
+			<h2 className="date-control-picker-date__heading">
+				{ translate( 'Date Range' ) }
+				<span> { translate( '(dd/mm/yyyy)' ) }</span>
```

Comment:
```suggestion
				<span>(dd/mm/yyyy)</span>
```
I don't think we'd need translate this.